### PR TITLE
Add Audit log support for store changes

### PR DESCRIPTION
**Pull request by ligangty**

### CODE REVIEW

File: src/main/java/org/commonjava/indy/service/repository/change/audit/StoreAuditManager.java [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+
+import static java.lang.String.format;
+
+@ApplicationScoped
+//@Startup
+public class StoreAuditManager
+{
+    private final Logger logger = LoggerFactory.getLogger( getClass() );
+
+    public static final String TABLE_AUDIT = "audittable";
```
Comment (sswguo) [inline src/main/java/org/commonjava/indy/service/repository/change/audit/StoreAuditManager.java:50]: Can we use another name without `table` ?  it's enough with just `audit` or `audit_log` or `repo_audit`.

**Review by sswguo — COMMENTED** [review]
Looks good except a minor thing. And let's see what's the records look like after tests.

File: src/main/java/org/commonjava/indy/service/repository/change/audit/StoreAuditManager.java (hunk 1 above)
Comment (ligangty) [reply src/main/java/org/commonjava/indy/service/repository/change/audit/StoreAuditManager.java]: Sounds good. Fixed.

**Comment by ligangty** [comment]
Merged and continue to testing.

**Pull request merged**